fusor/fusor
# Move RHV configuration from puppet to ansible

## Done
- Add task to RHV to configure and launch ansible 
- Refactor RHV deploy task
- Remove dead code related to puppet, since we don't use it any more
- Update OSE to be compatible with ansible 2.1

## TODO
- [x] Wait for the hosts to come up before running ansible (they reboot after provisioning)
- [x] Figure out packaging for [ansible-ovirt](http://github.com/fusor/ansible-ovirt)
- [x] Get ansible 2.1 onto Satellite

Thanks to @jmontleon for packaging help 


# To test
- Pull down changes in vm_env, fusor, and fusor-installer
- Switch to the ansible-ovirt branch in vm_env, and checkout this branch in fusor
- Run prep_checkouts.sh or manually clone https://github.com/fusor/ansible-ovirt
- vagrant up
- After vagrant up completes, ssh into the box, uninstall ansible, and run `easy_install ansible` to get 2.1+
- Everything after that should just work
